### PR TITLE
Update action button label on the editor sidebar.

### PR DESCRIPTION
**Pull request by mjuhasz**
This pull request updates the existing Publish button on the editor sidebar.

- Publish/Schedule steps are followed by a confirmation step now so the action button label on the editor sidebar indicates this with an ellipsis (…)
- Calendar dropdown is redundant so it's been removed

These changes are only visible if the 'post-editor/delta-post-publish-flow' feature flag is enabled.

fqejowpxyx

#### Existing behavior:

- Checkout branch and run *without* the feature-flag enabled: `make run`
- Load the editor and draft a post. Do not change the date.
- Verify that the look & feel and the behavior have not changed (hit Publish to publish your post)
- Change the date of the post, and verify that the look & feel and the behavior have not changed (hit Schedule to schedule your post)

#### New flow behind the feature flag:
- Checkout branch and run with the feature-flag enabled: ENABLE_FEATURES=post-editor/delta-post-publish-flow make run
- Load the editor and draft a post. Do not change the date.
- Verify that the Publish button has the ellipsis in its label, and it's functional
- Change the date of the post, and verify that the Schedule button has the ellipsis in its label, and it's functional

### CODE REVIEW

File: client/post-editor/editor-publish-button/index.jsx [hunk 1]
```diff
@@ -87,8 +88,16 @@ export class EditorPublishButton extends Component {
...
 				return this.props.translate( 'Update' );
 			case 'schedule':
+				if ( config.isEnabled( 'post-editor/delta-post-publish-flow' ) ) {
+					return this.props.translate( 'Schedule…',
+						{ comment: 'Button label on the editor sidebar - a confirmation step will follow' } );
+				}
 				return this.props.translate( 'Schedule' );
 			case 'publish':
+				if ( config.isEnabled( 'post-editor/delta-post-publish-flow' ) ) {
+					return this.props.translate( 'Publish…',
```
Comment (a8ci18n) [inline client/post-editor/editor-publish-button/index.jsx]: Hi! I've found a possible matching string that has already been translated **49** times:
`translate( 'Publish', { context: 'admin bar menu group label'} )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/editor-sidebar-indication)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

File: client/post-editor/editor-publish-button/index.jsx [hunk 2]
```diff
@@ -87,8 +88,16 @@ export class EditorPublishButton extends Component {
 			case 'update':
 				return this.props.translate( 'Update' );
 			case 'schedule':
+				if ( config.isEnabled( 'post-editor/delta-post-publish-flow' ) ) {
+					return this.props.translate( 'Schedule…',
```
Comment (a8ci18n) [inline client/post-editor/editor-publish-button/index.jsx]: Hi! I've found a possible matching string that has already been translated **75** times:
`translate( 'Schedule' )` *ES Score: 21*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/editor-sidebar-indication)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/post-editor/editor-ground-control/index.jsx [hunk 3]
```diff
@@ -285,6 +287,75 @@ export default React.createClass( {
 		}
 	},
 
+	renderGroundControlActionButtons: function() {
```
Comment (jonathansadowski) [inline client/post-editor/editor-ground-control/index.jsx]: Thanks for breaking this out into a separate method. Makes it easier to read!

File: client/post-editor/editor-publish-button/index.jsx (hunk 1 above)
Comment (yoavf) [inline client/post-editor/editor-publish-button/index.jsx]: A translator comment would have been better here!

**Pull request merged**